x-pack/plugins/alerting/server/task_runner/task_runner.ts

```diff
@@ -459,6 +459,8 @@ export class TaskRunner<
       actionsClient: await this.context.actionsPlugin.getActionsClientWithRequest(fakeRequest),
     });
 
+    let summaryActions = {};
```

Comment:
nit: maybe `summaryActionsState` would be more descriptive? It took me a bit to figure out why we needed to return this from the executionHandler.

Reply:
replaced with `executionHandlerRunResult` :)